dags/atd_signal_comms.py

```diff
@@ -0,0 +1,186 @@
+        auto_remove=True,
+        command=f"python atd-signal-comms/run_comm_check.py detector --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    dms_s3 = DockerOperator(
+        task_id="run_comm_check_dms",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python atd-signal-comms/run_comm_check.py digital_message_sign --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    battery_backup_s3 = DockerOperator(
+        task_id="run_comm_check_battery_backup",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python atd-signal-comms/run_comm_check.py cabinet_battery_backup --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    cameras_socrata = DockerOperator(
+        task_id="socata_pub_cameras",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python atd-signal-comms/socrata_pub.py camera --start {start_date} -v --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    detectors_socrata = DockerOperator(
+        task_id="socrata_pub_detectors",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python atd-signal-comms/socrata_pub.py detector --start {start_date} -v --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    dms_socrata = DockerOperator(
+        task_id="socrata_pub_dms",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python atd-signal-comms/socrata_pub.py digital_message_sign --start {start_date} -v --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    battery_backup_socrata = DockerOperator(
+        task_id="socrata_pub_battery_backup",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python atd-signal-comms/socrata_pub.py cabinet_battery_backup --start {start_date} -v --env {DEPLOYMENT_ENVIRONMENT}",
+        environment=env_vars,
+        tty=True,
+        force_pull=True,
+        mount_tmp_dir=False,
+        network_mode="bridge",
+    )
+
+    (
+        start_date
+        >> cameras_s3
+        >> detectors_s3
+        >> dms_s3
+        >> battery_backup_s3
+        >> cameras_socrata
+        >> detectors_socrata
+        >> dms_socrata
+        >> battery_backup_socrata
```

Comment:
i like this over the old way we'd write tags t1 >> t2 >> etc

Reply:
agreed!